examples/java/src/main/java/org/apache/ignite/example/table/RecordViewExample.java

```diff
@@ -43,80 +40,72 @@ public static void main(String[] args) throws Exception {
-                            + "balance       DOUBLE)"
-            );
-        }
-
-        //--------------------------------------------------------------------------------------
-        //
-        // Creating a client to connect to the cluster.
-        //
-        //--------------------------------------------------------------------------------------
-
-        System.out.println("\nConnecting to server...");
-
         try (IgniteClient client = IgniteClient.builder()
                 .addresses("127.0.0.1:10800")
                 .build()
         ) {
-            //--------------------------------------------------------------------------------------
-            //
-            // Creating a record view for the 'accounts' table.
-            //
-            //--------------------------------------------------------------------------------------
-
-            RecordView<Tuple> accounts = client.tables().table("accounts").recordView();
-
-            //--------------------------------------------------------------------------------------
-            //
-            // Performing the 'insert' operation.
-            //
-            //--------------------------------------------------------------------------------------
-
-            System.out.println("\nInserting a record into the 'accounts' table...");
-
-            Tuple newAccountTuple = Tuple.create()
-                    .set("accountNumber", 123456)
-                    .set("firstName", "Val")
-                    .set("lastName", "Kulichenko")
-                    .set("balance", 100.00d);
-
-            accounts.insert(null, newAccountTuple);
-
-            //--------------------------------------------------------------------------------------
-            //
-            // Performing the 'get' operation.
-            //
-            //--------------------------------------------------------------------------------------
-
-            System.out.println("\nRetrieving a record using RecordView API...");
-
-            Tuple accountNumberTuple = Tuple.create().set("accountNumber", 123456);
-
-            Tuple accountTuple = accounts.get(null, accountNumberTuple);
-
-            System.out.println(
-                    "\nRetrieved record:\n"
-                            + "    Account Number: " + accountTuple.intValue("accountNumber") + '\n'
-                            + "    Owner: " + accountTuple.stringValue("firstName") + " " + accountTuple.stringValue("lastName") + '\n'
-                            + "    Balance: $" + accountTuple.doubleValue("balance"));
-        } finally {
-            System.out.println("\nDropping the table...");
-
-            try (
-                    Connection conn = DriverManager.getConnection("jdbc:ignite:thin://127.0.0.1:10800/");
-                    Statement stmt = conn.createStatement()
-            ) {
-                stmt.executeUpdate("DROP TABLE accounts");
+            try {
```

Comment:
Please do not change unrelated examples. This example should be reverted. If you want to overcome the broken compatibility issue (https://issues.apache.org/jira/browse/IGNITE-27280) then please use the latest Ignite 3 build from `main`.